v2/bin/hdxworkbench.py

```diff
@@ -124,3 +151,44 @@
 
     # This temperature tends to sit around 15% MC acceptance rate, which seems to be good.
     sampler.run(args.nsteps, 1, write=True)
+
+    files = [f for dr, ds, files in os.walk(args.outputdir) for f in files if f.endswith('.dat')]
+    control_files = []
+    ligand_files = []
+    for f in files:
+        if args.control in f:
+            control_files.append(f)
+        elif args.ligand in f:
+            ligand_files.append(f)
+    os.chdir(args.outputdir)
```

Comment:
Generally `os.chdir` should be avoided since it changes the working directory for the entire process. Consider using full paths to output files instead.